app/src/main/res/values/strings.xml

```diff
@@ -139,4 +148,6 @@
     <string name="allow_location">Allow Location</string>
     <string name="location_pin_image">Location pin image.</string>
     <string name="button_for_sending_message">Button for sending message.</string>
+    <string name="play_audio">Play Audio</string>
+    <string name="record_audio">Record</string>
 </resources>
```

Comment:
I stopped commenting after a few, I think something's might be wrong with Cirrus, it's really weird that all strings are unused...